Resources/public/js/views/universaldiscovery/ez-universaldiscoveryselectedview.js

```diff
@@ -42,6 +43,23 @@ YUI.add('ez-universaldiscoveryselectedview', function (Y) {
         },
 
         /**
+         *
+         * @method _setConfirmButtonEnabled
+         * @protected
+         */
+        _setConfirmButtonEnabled: function (contentStruct) {
+            var isAlreadySelected = Y.bind(this.get('isAlreadySelected'), this),
+                isSelectable = Y.bind(this.get('isSelectable'), this);
```

Comment:
I would not bind those functions to the selected view. If you want to bind those functions to a context, that should the universal discovery view (that's already the case for `isAlreadySelected` and maybe we should do the same for `isSelectable`)